data/assets/examples/rotation/timelinerotation/no-interpolate.asset

```diff
@@ -0,0 +1,49 @@
+-- No Interpolation
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The rotation of
```

Comment:
Maybe mention the "no interpolation" part in the first sentence already?  So that it explains why it is different to the "Basic" example